x/leverage/keeper/internal_test.go

```diff
@@ -0,0 +1,63 @@
+package keeper
```

Comment:
maybe use `keeper_test` here instead?

Reply:
This has to be `package keeper` so it can access unexported methods. (`keeper_test` would have to import `keeper`)

But since the filename ends in `_test.go` it doesn't build outside of tests, and it's not accessible to packages other than `keeper_test` (even `x/leverage/simulation` can't use `NewTestKeeper`).

This seemed to be the way to do it, to allow for testing of unexported methods using the test suite.

Reply:
ACK 👍